html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <form action="https://example.com/">
+    <p>Email
+    <div>
+      <input type="text" name="email" id="email">
+      <label for="email"></label>
+    </div>
+    
+    <p>Why is coding cool?
```

Comment:
label needed here, and make sure to include the proper `for` and `id` attributes on the label and input

Reply:
![html-forms revised](https://user-images.githubusercontent.com/90783131/151492656-6dc4341e-ca1a-4bd2-a143-c4b77cfb98d4.gif)